build.gradle

```diff
@@ -86,4 +86,12 @@ subprojects {
     test {
         useJUnitPlatform()
     }
+
+    compileJava {
+        options.encoding = "UTF-8"
+    }
+
+    compileTestJava {
+        options.encoding = "UTF-8"
+    }
```

Comment:
Might be a good idea to make another PR for these changes no? Ideally working on single issue keeps PRs focussed and meaningful.

Reply:
I am not sure about this, I already thought about making a different PR for it but also thought thats just extra work for no benefit actually

Reply:
It's just two lines of code, either way it's out of scope for this PR. Would appreciate if you can address this in new PR.

Reply:
I don't really see the reason, it's not that important to have a separate PR in my opinion
going to wait what other say about this

Reply:
i get ankis point but i also dont mind too much.
at the end of the day what counts is what ends in master 🤷

Reply:
idk it's the end of the world you can very well have all your code in one PR, but ig the idea behind suggestion was to promote more atomic PRs. Whatever we choose do reflect across project in future PRs as well, although no significant impact but just becomes a habbit in general.

Reply:
I just want to avoid people being too annoyed by pure bureaucracy so that the end result is that they wont fix smaller issues as they read them because "damn, im too lazy to put these 3 smaller typos here in a separate PR".

I rather have people fix small shit as they read it than not having the improvements at all. Thats my POV and I have witnessed this issue several times in other (bigger) projects.

But I ofc also agree that in a perfect world, it should be in a separate PR.

Reply:
very well, if you say  so.